GenumKit/Parsers/AssetsCatalogParser.swift

```diff
@@ -12,7 +12,8 @@ public final class AssetsCatalogParser {
 
   public init() {}
 
-  public func addImageName(name: String) -> Bool {
+  @discardableResult
+  public func addImageName(_ name: String) -> Bool {
```

Comment:
I went back and fourth on this one, I think because of the collection is actually `imageNames` I have to confess I'm still a little unclear on if there is an objective differentiator between `addImage()` `add(image:)` 😬 I agree though, `addImage(named name: String)` is best for api continuity.